src/async-wrap.h

```diff
@@ -117,7 +117,7 @@ class AsyncWrap : public BaseObject {
 
   inline double get_trigger_id() const;
 
-  void AsyncReset(bool silent = false);
+  void AsyncReset(double eid = 0, bool silent = false);
```

Comment:
Sounds good. There are benefits for using either, but I was on the fence about which to always default to. If everyone else thinks `-1` is the better value then I'll be happy to go with that.